conf/redpanda.yaml

```diff
@@ -46,6 +46,23 @@ redpanda:
   # not recomended for production use
   developer_mode: true
 
+  enable_proxy: true
+
+pandaproxy:
+  # Pandaproxy transport
+  pandaproxy_api:
+    address: "0.0.0.0"
+    port: 8082
+
+  admin_api_doc_dir: ../../../pandaproxy/admin-api-doc
```

Comment:
should we symlink them inside the conf dir.

Reply:
I've made the default sensible and fixed packaging